docs/scripts_md/File.md

```diff
@@ -0,0 +1,178 @@
+# NAME
+
+NeuroDB::File -- Provides an interface to the MRI file management subsystem of
+LORIS
+
+# SYNOPSIS
+
+    use NeuroDB::File;
+    use NeuroDB::DBI;
+
+    my $dbh = NeuroDB::DBI::connect_to_db();
+
+    my $file = NeuroDB::File->new(\$dbh);
+
+    my $fileID = $file->findFile('/path/to/some/file');
+    $file->loadFile($fileID);
+
+    my $acquisition_date = $file->getParameter('acquisition_date');
+    my $parameters_hashref = $file->getParameters();
+
+    my $coordinate_space = $file->getFileDatum('CoordinateSpace');
+    my $filedata_hashref = $file->getFileData();
+
+
+    # less common to use methods, available mainly for register_db...
+    my $dbh_copy = $file->getDatabaseHandleRef();
+
+    $file->loadFileFromDisk('/path/to/some/file');
+    $file->setFileData('CoordinateSpace', 'nonlinear');
+    $file->setParameter('patient_name', 'Larry Wall');
+
+    my $parameterTypeID = $file->getParameterTypeID('patient_name');
+
+# DESCRIPTION
+
+This class defines a BIC MRI (or related) file (minc, bicobj, xfm,
```

Comment:
why a BIC MRI?:)

Reply:
It was what was documented in the original file so I did not change it. I'll remove the BIC.

Reply:
I know; just gives us an idea of how far Loris-MRI has advanced (from only supporting BIC MRIs' to MRIs from anywhere now):)